sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -1330,7 +1330,6 @@ static bool compatibleWithDevice(RTDeviceBinaryImage *BinImage,
 
 kernel_id ProgramManager::getSYCLKernelID(const std::string &KernelName) {
   std::lock_guard<std::mutex> KernelIDsGuard(m_KernelIDsMutex);
-
```

Comment:
Though removing these lines aren't that disruptive, I would prefer they stay just to not take focus away from the intent of this commit. Not a stopper though.

Reply:
Ok, I'll return them.